### PR TITLE
add 1.6.0 en api ref and release

### PR DESCRIPTION
**Pull request by Nero-Hu**

### CODE REVIEW

File: Fastboard/EN-API Reference.md [hunk 1]
```diff
@@ -134,6 +146,22 @@ public class FastUserPayload {
...
+    private final int preloadCount;
+}
+```
+
+The auxiliary configuration class of the `Fastboard` object, including the following attributes:
+
+- `enablePreload`: Boolean. Whether to enable whiteboard preloading:
+  - `true`: Enable whiteboard preloading. Enabling this can improve whiteboard loading speed.
+  - `false`: (Default) Disable whiteboard preloading.
+- `preloadCount`: int. The number of WebViews to preload, defaults to `0`. The actual effect of this attribute is limited by the memory capacity. You need to adjust the settings appropriately according to the actual situation.
```
Comment (atovpeko) [inline Fastboard/EN-API Reference.md:163]:
```suggestion
- `preloadCount`: Integer. The number of WebViews to preload, defaults to `0`. The actual effect of this attribute is limited by the memory capacity. You need to adjust the settings appropriately according to the actual situation.
```

File: Fastboard/EN-API Reference.md [hunk 2]
```diff
@@ -389,12 +421,68 @@ Insert and play audio and video in the whiteboard sub-window.
...
+```java
+public class DocPage {
+    private String src;
+    private String preview;
+    private Double width;
+    private Double height;
+}
+```
+
+A class for setting the page settings for inserting documents. Includes the following attributes
```
Comment (atovpeko) [inline Fastboard/EN-API Reference.md:449]:
```suggestion
A class for configuring the page settings for inserting documents. Includes the following attributes:
```

File: Fastboard/EN-API Reference.md [hunk 3]
```diff
@@ -389,12 +421,68 @@ Insert and play audio and video in the whiteboard sub-window.
...
+
+- `src`: String. The address of the image or dynamic PPT page.
+  - Image: URL address, which can be generated by yourself or by the document conversion function. You can get the address from the image field returned by the query conversion task progress. Make sure the App client can access the URL, otherwise the image cannot be loaded normally.
+  - Dynamic PPT page: URI address generated by the document conversion function.
+- `preview`: String. (Optional) URL address of the image or dynamic PPT preview image. The URL address of the dynamic PPT preview image can be obtained from the preview field returned by the query document conversion progress. After passing in, the preview image will be displayed on the left side of the App. Make sure the App client can access the URL, otherwise the preview image cannot be loaded normally.
+- `width`: int. The width of the inserted document in the whiteboard (px).
```
Comment (atovpeko) [inline Fastboard/EN-API Reference.md:455]:
```suggestion
- `width`: Integer. The width of the inserted document in the whiteboard (px).
```

File: Fastboard/EN-API Reference.md [hunk 4]
```diff
@@ -389,12 +421,68 @@ Insert and play audio and video in the whiteboard sub-window.
...
+- `height`: int. The height of the inserted document in the whiteboard (px).
```
Comment (atovpeko) [inline Fastboard/EN-API Reference.md:456]:
```suggestion
- `height`: Integer. The height of the inserted document in the whiteboard (px).
```

File: Fastboard/EN-API Reference.md [hunk 5]
```diff
@@ -786,6 +874,74 @@ The default display state of the toolbar is related to the device. It is expande
...
+
+```java
+public static void setLogger(Logger logger)
+```
+
+Set custom logger.
+
+**Parameters**
+
+- `logger`: The logger instance. If you need to customize the logger, you need to pass in an implementation class of the `Logger` interface, see [`Logger`](#logger); passing in `null` will use the system default logger.
```
Comment (atovpeko) [inline Fastboard/EN-API Reference.md:891]: I would change "will use" to "will result in using" for better grammar

File: Fastboard/EN-API Reference.md [hunk 6]
```diff
@@ -786,6 +874,74 @@ The default display state of the toolbar is related to the device. It is expande
...
+
+- `msg`: String. Log message. 
+
+### info
+
+```java
+static void info(String msg)
+```
+
+Record a info log.
```
Comment (atovpeko) [inline Fastboard/EN-API Reference.md:915]:
```suggestion
Record an info log. 
```

File: Fastboard/EN-API Reference.md [hunk 7]
```diff
@@ -786,6 +874,74 @@ The default display state of the toolbar is related to the device. It is expande
...
+
+- `msg`: String. Log message. 
+
+### error
+
+```java
+static void error(String msg)
+```
+
+Record a error 级别日志.
```
Comment (atovpeko) [inline Fastboard/EN-API Reference.md:939]: 1. a --> an
2. The Chinese part still needs to be translated, I assume?

File: Fastboard/EN-Release Notes.md [hunk 8]
```diff
@@ -3,6 +3,61 @@
...
+**New Whiteboard Tools**
+
+This release adds the following new whiteboard tools: 
+
+- Adds pencil eraser tool FastAppliance.PENCIL_ERASER to erase local pencil strokes.
+- Adds grab tool FastAppliance.HAND to move the view.
+
+**Whiteboard Logging**
+
+Adds the `FastLogger` class to provide a flexible whiteboard runtime log recording tool, which can easily record `debug`, `info`, `warn`, and `error` four different levels of log information. You can call the `setLogger` method to customize the logger to adapt to different logging needs, or you can use the system `Log` class to log.
```
Comment (atovpeko) [inline Fastboard/EN-Release Notes.md:29]:
```suggestion
Adds the `FastLogger` class to provide a flexible whiteboard runtime log recording tool, which can easily record four different levels of log information: `debug`, `info`, `warn`, and `error` . You can call the `setLogger` method to customize the logger to adapt to different logging needs, or you can use the system `Log` class to log.
```

File: Fastboard/EN-Release Notes.md [hunk 9]
```diff
@@ -3,6 +3,61 @@
...
+
+#### Improvements
+
+- When calling the `Fastroom.insertImage` method to insert an image, the image will be centered in the current view by default.
+- The full room state callback roomState after joining the room is disabled. Clients need to call `Room.getRoomState()` to get the full `RoomState`.
```
Comment (atovpeko) [inline Fastboard/EN-Release Notes.md:34]: The full room state callback roomState after joining the room is disabled - > this sentence is slightly awkward, can it be rephrased for readability?

File: Fastboard/EN-Release Notes.md [hunk 10]
```diff
@@ -3,6 +3,61 @@
...
+  - `HAND`
+- `FastLogger`
+  - `setLogger`
+  - `Logger`
+  - `warn`
+  - `debug`
+  - `info`
+  - `error`
+
+**Depreacated**
```
Comment (atovpeko) [inline Fastboard/EN-Release Notes.md:56]:
```suggestion
**Deprecated**
```

**Review by atovpeko — CHANGES_REQUESTED** [review]

File: Fastboard/EN-Release Notes.md (hunk 9 above)
Comment (Nero-Hu) [reply Fastboard/EN-Release Notes.md:34]: Please check if this version is better.
The room state callback, `roomState`, which is triggered after joining a whiteboard room, is disabled. You need to call `Room.getRoomState()` to get the complete `RoomState`.

**Pull request merged**